browser-test/src/header.test.ts

```diff
@@ -1,56 +1,59 @@
-import {test, expect} from '@playwright/test'
+import {test, expect} from './support/civiform_fixtures'
 import {
-  createTestContext,
   loginAsTestUser,
   validateScreenshot,
   validateAccessibility,
+  logout,
 } from './support'
 
-test.describe('Header', () => {
-  const ctx = createTestContext(/* clearDb= */ false)
-
-  test('Not logged in, guest mode enabled', async () => {
-    const {page} = ctx
-    await validateScreenshot(
-      page.getByRole('navigation'),
-      'not-logged-in-guest-mode-enabled',
-    )
-  })
-
-  // TODO(#4360): add a "Not logged in, guest mode disabled" test once we
-  // can get to the programs page without logging in, for an entity without
-  // guest mode.
-
-  test('Logged in', async () => {
-    const {page} = ctx
-    await loginAsTestUser(page)
-    await validateScreenshot(page.getByRole('navigation'), 'logged-in')
+test.describe('Header', {tag: ['@uses-fixtures']}, () => {
```

Comment:
tag `@uses-fixtures` will be used to mark which files have been migrated over and will go away once everything is done.

Reply:
Smart!